WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -172,5 +172,18 @@ private extension CardReaderSettingsViewController {
 
     func configureNavigation() {
         title = NSLocalizedString("Card Readers", comment: "Card reader settings screen title")
+        // Adding these button just for testing
```

Comment:
This is definitely not something we want in the final UI. But, it allows us to have something in the UI that allows testing,